.github/workflows/maven.yml

```diff
@@ -16,5 +16,14 @@ jobs:
       with:
         distribution: 'adopt'
         java-version: '11'
-    - name: Build with Maven
-      run: mvn clean verify -Pbuild-individual-bundles -Dskip.ui-tests=true
+    - name: Cache local Maven repository
+      uses: actions/cache@v3
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
+    - name: Run build
+      uses: GabrielBB/xvfb-action@v1
+      with:
```

Comment:
Just curious - where all the strange action names are coming from?

Reply:
See https://github.com/GabrielBB/xvfb-action for example.

Reply:
@iloveeclipse Github actions are named by their creators.

Reply:
@iloveeclipse Github actions are named by their creators and AFAIK GH does not have a standard action for framebuffer setup.

Reply:
... and how do you discover available actions? Is there some kimd of directory/catalog or the like?

Reply:
There is https://github.com/marketplace?type=actions or you can write your own or "google" for a certain use-case most of the time your not the first with a certain problem.